Examples/ComplexVaporExample/Resources/Views/kitten.leaf

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <title>Kittens</title>
+  <style type="text/css">
+    table, th, td {
+      border: 1px solid black;
+    }
+    th, td {
+      padding: 8px;
+    }
+    h1, h2, th, td, label {
+      font-family: "Trebuchet MS", Helvetica, sans-serif;
+    }
+  </style>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+</head>
+<body>
+<h1>#(name) 🐱</h1>
+<table>
+<tr>
+  <th>Color</th>
+  <td>#(color)</td>
+</tr>
+<tr>
+  <th>Favorite Food</th>
+  <td>#(favoriteFood)
+  </td>
+</tr>
+</table>
+<br><br>
+<h2>Edit</h2>
+<label for="favoriteFood">New Favorite Food</label>
+<select id="favoriteFood" name="favoriteFood">
+  <option value="" selected disabled hidden>Select</option>
+  <option value="salmon">salmon</option>
+  <option value="turkey">turkey</option>
+  <option value="chicken">chicken</option>
+  <option value="tuna">tuna</option>
+  <option value="beef">beef</option>
+</select>
+</div>
+<div>
+<button id="save-btn">Save</button>
+<br>
+<br>
+</div>
+<br><br><br>
+<button id="delete-btn">Delete kitten</button>
+<br>
+<br>
+<a href="/">Home</a>
+<div id="error-info"></div>
+
+<script type="text/javascript">
```

Comment:
TIL (or maybe I used to know this, but I'd long forgotten) that you can't use a form to send a DELETE or PATCH request, so I had to break out the jQuery here to exercise those code paths. sigh.